actions/admin_actions.jsx

```diff
@@ -5,7 +5,7 @@ import * as AdminActions from 'mattermost-redux/actions/admin';
 import * as UserActions from 'mattermost-redux/actions/users';
 import {Client4} from 'mattermost-redux/client';
 
-import {clientLogout} from 'actions/global_actions.jsx';
+import {emitUserLoggedOutEvent} from 'actions/global_actions.jsx';
```

Comment:
`clientLogout` was basically `emitUserLoggedOutEvent` without the `Client4.logout` (odd name choice!), but this created multiple entry points for what essentially amounted to the same function. I've tested the various changes here and continue to get logged out.